skrf/tests/test_network.py

```diff
@@ -639,8 +640,15 @@ def test_noise_deembed(self):
           self.assertTrue(abs(gamma_opt_rb.s[0,0,0] - gamma_opt_set) < 1.e-1, 'nf not retrieved by noise deembed')
 
 
+    def test_se2gmm2se_mag(self):
+        ntwk4 = rf.Network(os.path.join(self.test_dir, 'cst_example_4ports.s4p'))
+        ntwk4t = deepcopy(ntwk4)
```

Comment:
Why do you make a deepcopy of ntw4?

Reply:
se2gmm and gmm2se modify the network object directly, so I want a deepcopy around so that there is an unmodified original copy to compare against. Alternatively, to eliminate the deepcopy, we could just create another object from the same input file.